cloudsync/provider.py

```diff
@@ -408,19 +408,24 @@ def split(self, path):
             return self.sep, path[index + 1:]
         return path[:index], path[index+1:]
 
-    def normalize_path(self, path: str):
-        """Used internally for comparing paths in a case and sep insensitive manner, as appropriate."""
-        norm_path = path.rstrip(self.sep)
-        if self.sep in ["\\", "/"]:
-            parts = re.split(r'[\\/]+', norm_path)
-        else:
-            parts = re.split(r'[%s]+' % self.sep, norm_path)
+    def normalize_path(self, path: str, pretty_leaf: bool = False):
```

Comment:
pretty_leaf should be called use_display_case, "display path" is the term used to connote the capitalization chosen by the user for the leaf node

Reply:
done.